connectors/hugging-face/element-templates/hugging-face-connector.json

```diff
@@ -0,0 +1,225 @@
+  },
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:Task"
+  ],
+  "elementType": {
+    "value": "bpmn:ServiceTask"
+  },
+  "groups": [
+    {
+      "id": "authentication",
+      "label": "Authentication"
+    },
+    {
+      "id": "input",
+      "label": "Payload"
+    },
+    {
+      "id": "output",
+      "label": "Response mapping"
+    },
+    {
+      "id": "errors",
+      "label": "Error handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:http-json:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "id": "authenticationType",
+      "value": "bearer",
+      "type": "Hidden",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      }
+    },
+    {
+      "label": "Hugging Face API key",
+      "group": "authentication",
+      "feel": "optional",
+      "description": "Your Hugging Face API key",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.token"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Model",
+      "group": "input",
+      "feel": "optional",
+      "value": "gpt2",
+      "description": "Desired model name",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "modelName"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "id": "inputType",
+      "label": "Input type",
+      "group": "input",
+      "description": "Whether to use simplified or fine-tuned input",
```

Comment:
we should link here the https://huggingface.co/docs/api-inference/detailed_parameters
and rename fine-tuning with advamced input configuration or sth. similar

Reply:
Removed dropdown for now as discussed offline